akvo/rsr/models/indicator.py

```diff
@@ -196,7 +196,7 @@ def children_aggregate_percentage(self):
         """
         if self.measure == '2' and self.is_parent_indicator() and \
                 self.result.project.aggregate_children and \
-                any([ind.result.project.aggregate_to_parent for ind in self.child_indicators.all()]):
+                any(self.child_indicators.values_list('result__project__aggregate_to_parent', flat=True)):
```

Comment:
The query that runs looks something like this:

```
SELECT "rsr_project"."aggregate_to_parent" FROM "rsr_indicator" INNER JOIN "rsr_result" ON ( "rsr_indicator"."result_id" = "rsr_result"."id" ) INNER JOIN "rsr_project" ON ( "rsr_result"."project_id" = "rsr_project"."id" ) WHERE "rsr_indicator"."parent_indicator_id" = 23156 ORDER BY "rsr_indicator"."order" ASC, "rsr_indicator"."id" ASC
```

Prefetching the `child_indicators` doesn't seem to really change the query in any way.